tests/dummy/app/templates/application.hbs

```diff
@@ -6,10 +6,10 @@
   </div>
   <div class="input-wrapper {{if this.isJSONValid "valid" "invalid"}}">
     <h2>Paste JSON below:</h2>
-    <Textarea
+    <textarea aria-label="json"
```

Comment:
ember-template-lint wanted the label and for this to be a plain HTML `<textarea>`